src/libs/actions/App.js

```diff
@@ -211,19 +211,10 @@ function setUpPoliciesAndNavigate(session) {
         return;
     }
     if (!isLoggingInAsNewUser && exitTo) {
-        if (Navigation.isDrawerRoute(exitTo)) {
-            // The drawer navigation is only created after we have fetched reports from the server.
-            // Thus, if we use the standard navigation and try to navigate to a drawer route before
-            // the reports have been fetched, we will fail to navigate.
-            Navigation.isDrawerReady()
-                .then(() => {
-                    // We must call dismissModal() to remove the /transition route from history
-                    Navigation.dismissModal();
-                    Navigation.navigate(exitTo);
-                });
-            return;
-        }
-        Navigation.isNavigationReady()
```

Comment:
I think we might still want to have the isNavigationReady check here, there might be a case where the navigation is not ready but the drawer is.

Reply:
I think we would do something like this, or make a new / update an existing `isXyzReady` to check if navigation and the drawer are both ready
```
            Navigation.isNavigationReady().then(() => Navigation.isDrawerReady().then(() => //navigate));
```